DependencyInjection/Configuration.php

```diff
@@ -107,7 +107,11 @@ public function getConfigTreeBuilder()
                                 return $v;
                             })
                         ->end()
-                        ->defaultValue(class_exists('Doctrine\ORM\EntityManager'))->end()
+                        ->defaultValue(class_exists('Doctrine\ORM\EntityManager'))
+                    ->end()
+                    ->scalarNode('service_naming_strategy')
```

Comment:
Wouldn't having the alias defined in the XML file mean that you need to override the `jms_di_extra.service_naming_strategy` service in order to supply your own naming strategy service.

Overriding (with a custom compiler pass) is a much less comfortable extension approach compared to declaring a different service in the config.